plugin/util.go

```diff
@@ -45,6 +46,24 @@ func toEnvironment(args Args) string {
 	return "production"
 }
 
+// helper function determines the target environment.
+func toEnvironmentId(args Args) string {
+	if v := args.EnvironmentId; v != "" {
+		return toEnvironmentEnum(v)
+	}
+	// default environment if none specified.
+	return "production"
+}
+
+// helper function determines the target environment.
+func toEnvironmentType(args Args) string {
+	if v := args.EnvironmentType; v != "" {
+		return toEnvironmentEnum(v)
+	}
+	// default environment if none specified.
+	return "production"
+}
```

Comment:
both toEnvironmentId and toEnvironmentType functions currently reuse the logic of toEnvironmentEnum. Since they don’t seem to need the enum logic (as they are IDs or types), you might want to make them return args.EnvironmentId or args.EnvironmentType directly instead of converting them via toEnvironmentEnum